python.Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM jenkins/jnlp-slave:alpine
 
 ENV SCANNER_VERSION 3.3.0.1492
 ENV DOCKER_VERSION=18.09.5
+ENV HELM_VERSION=3.1.0
```

Comment:
We are currently using helm version 3.0.3, I'm not sure if this is compatible...

Reply:
Makes sense! I have been using 3.1.0 already, but we should aim to use our tried and tested version and update this once we're confident there aren't any problems.